pkg/api/controller_test.go

```diff
@@ -6628,3 +6628,42 @@ func TestController_GetLicense(t *testing.T) {
 		require.NotNil(t, resp.JSON501, "expected HTTP-501 response, got %v", resp.StatusCode())
 	})
 }
+
+func TestController_ImportStart_Disabled(t *testing.T) {
+	// Override config
+	p := t.TempDir()
+	viper.Set(config.BlockstoreTypeKey, block.BlockstoreTypeLocal)
+	viper.Set("blockstore.local.path", p)
+	viper.Set("blockstore.local.import_enabled", false)
+
+	// Create server
+	clt, deps := setupClientWithAdmin(t)
+	ctx := context.Background()
+
+	// Define repo
+	repo := testUniqueRepoName()
+	// Create repo
+	_, err := deps.catalog.CreateRepository(ctx, repo, config.SingleBlockstoreID, onBlock(deps, "bucket/prefix"), "main", false)
+	require.NoError(t, err)
```

Comment:
- remove redundant comments or specify one that say that we create a repo with unique repo
- verify the status code in the create repo response (the better will be the response.JSON201)